.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,43 @@
+name: Bug report
```

Comment:
![image](https://github.com/user-attachments/assets/5b68a6df-31e5-40e3-b78a-375ea4d769f7)

Reply:
It doesn't show that anymore.